src/prefect/server/events/triggers.py

```diff
@@ -83,18 +83,39 @@ async def evaluate(
     each automation, and will fire the associated action if it has met the threshold."""
     automation = trigger.automation
 
+    logging_context = {
```

Comment:
The additional logging here matches the very helpful diagnostic logging from Prefect Cloud that was instrumental in understanding this bug.